### DISCUSSION TITLE
Localized map

### DISCUSSION THREAD

**Discussion by jbelien** [EqualStreetNames/equalstreetnames#772] [Ideas]
There is a [Mapbox plugin](https://github.com/mapbox/mapbox-gl-language) to have a map that displays labels in a specific language.
Unfortunately, it doesn't support NL at the moment (see [documentation](https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v7/#name-fields)).

(it also doesn't support `dark-v10` at the moment, only `dark-v9` - see [documentation](https://github.com/mapbox/mapbox-gl-language#supported-styles)).

**Comment by 1ec5** [comment]
Unfortunately the [Mapbox Streets source](https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v8/#name-fields) doesn’t have a dedicated Dutch field either, so it’s only possible for GL JS to show streets in the local language, French, or English if you show roads directly from Mapbox Streets vector tiles. However, if you’re showing roads from another source, such as an Overpass query, then you could use gl-language as long as you override the `supportedLanguages` option to include `nl` and override `getLanguageField` to use `name:` as the key prefix instead of `name_`.

**Comment by jbelien** [comment]
Thanks a lot for the feedback @1ec5 !

**Comment by jbelien** [comment]
Meanwhile I discovered that base maps from @maptiler are available in FR and NL (and are more up-to-date) so it's maybe best just to switch to those base maps (see https://www.maptiler.com/cloud/)
